installing/installing_nutanix/preparing-to-install-on-nutanix.adoc

```diff
@@ -1,14 +1,21 @@
 :_mod-docs-content-type: ASSEMBLY
 [id="preparing-to-install-on-nutanix"]
-= Preparing to install on Nutanix
+= Installation methods
```

Comment:
I won't hold up the merge over it, but I think the heading was better the way it was. I'd consider changing it (maybe back) in a future PR, for the following reasons:

- "Installation methods" doesn't tell you that this is Nutanix-specific. See this minimalism principle for descriptive headings: https://redhat-documentation.github.io/supplementary-style-guide/#_principle_3_titles_and_headings
- The left-hand TOC (in the topic map) still says "Preparing to install on Nutanix"
- The "Preparing to..." title seems to be consistent with all of the titles of the other providers